synapse/lib/webapp.py

```diff
@@ -62,19 +62,19 @@ def post(self, *args):
 
         boss.initJob( task=(func,args,kwargs), ondone=self._onJobDone )
 
-    def _onJobDone(self, job):
-
-        # a bit of "json friendlyness"
-        ret = {'status':'ok'}
-
+    def format_job_response(self, job):
+        """Format job results into a standard response envelope."""
+        ret = {'status': 'ok'}
         err = job[1].get('err')
-        if err != None:
+        if err is not None:
```

Comment:
i avoid the use of "is not" even for singleton objects like None because i've encountered C extensions which actually instantiate an additional instance of None for their returns which causes "is" (or is not) to fail since the actual test behind "is" requires that both reference the same instance.  Please switch back to !=